### PR TITLE
Push notifications integration tests for Android

### PR DESCRIPTION
**Pull request by b1zzu**

### CODE REVIEW

File: test/push/index.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+  it('send and recive test notification', async function () {
+
+    // register the app to the UPS server
+    const error = await client.executeAsync((config, done) => {
+      const { agPush, agApp } = window.aerogear;
+      const { init } = agApp;
+      const { PushRegistration } = agPush;
+
+      document.addEventListener("deviceready", () => {
```
Comment (jhellar) [inline test/push/index.js]: It should not be needed to wait for `deviceready` in specific tests, this should be already done here: https://github.com/jhellar/aerogear-integration-tests/blob/master/util/init.js#L14-L26

**Comment by jhellar** [comment]
👍 looks good

File: test/push/index.js (hunk 1 above)
Comment (b1zzu) [reply test/push/index.js]: removed

**Comment by b1zzu** [comment]
@jhellar for me it is ready

File: docker-compose.yml [hunk 2]
```diff
@@ -31,6 +31,22 @@ services:
       PGDATABASE: aerogear_mobile_metrics
     depends_on:
       - postgres
+  upsdb:
```
Comment (jhellar) [inline docker-compose.yml]: can we try to share db for both metrics and ups?

**Comment by b1zzu** [comment]
Moved

**Pull request closed without merging**